src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public static String getRandomColor() {
```

Comment:
Don't use static methods in your solution. Instead, create an instance of the class to call its methods. Static methods are generally considered bad practice unless they are utility functions, which is not the case here.